src/seedu/addressbook/AddressBook.java

```diff
@@ -310,6 +314,7 @@ private static void setupDefaultFileForStorage() {
      * Returns true if the given file path is valid.
      * A file path is valid if it has a valid parent directory as determined by {@link #hasValidParentDirectory}
      * and a valid file name as determined by {@link #hasValidFileName}.
+     * @param filePath to the file
```

Comment:
Note that in javadoc, there should be an empty line between the method description and @ annotation.